src/commands/promote.js

```diff
@@ -0,0 +1,73 @@
+const Command = require('../base');
+const { flags: flagUtils } = require('@oclif/command');
+
+class PromoteCommand extends Command {
+  /**
+   * Runs the promote command for a specific version of a package that has already been built
+   *
+   * @async
+   * @returns {Promise} The result or error
+   */
+  async run() {
+    const { flags, args } = this.parse(PromoteCommand);
+    const { env } = args; // package is reserved so we don't try to destructure it
```

Comment:
> // package is reserved so we don't try to destructure it

huh? I'm not sure what you mean? if you're talking about the naming, you can always rename while destructuring like so:

```suggestion
    const { env, package: packageInfo } = args;
```

Reply:
I think it is talking about the naming. Maybe @fritzmonkey can confirm. Since the all other existing commands do it this way, I decided to stay consistent. I agree that it is clearer if we just rename while destructuring. I can go and change this for all commands